typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,78 @@
+interface amazonItem {
```

Comment:
By convention user defined types/interfaces should be capitalized (as opposed to primitive types). So I'd go with: AmazonItem